managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -480,7 +481,7 @@ public void operationComplete(ManagedCursorInfo info, Stat stat) {
                     // Need to proceed and read the last entry in the specified ledger to find out the last position
                     log.info("[{}] Consumer {} meta-data recover from ledger {}", ledger.getName(), name,
                             info.getCursorsLedgerId());
-                    recoverFromLedger(info, callback);
+                    recoverFromLedger(info, callback, 2);
```

Comment:
why hardcode 2 here?

Reply:
Thanks for the comment. The modification has changed. see https://github.com/apache/pulsar/pull/18123#discussion_r1000713957. Could you take a look again?